ui/audiopage.cpp

```diff
@@ -33,11 +33,6 @@ AudioPage::AudioPage(QWidget *parent) : QWidget(parent)
     setupUI();
 }
 
-AudioPage::~AudioPage()
-{
-    delete this;
```

Comment:
`delete this` in a constructor should never be needed. This (and the other similar destructors) were causing double frees.